src/test/java/org/jboss/hal/testsuite/test/configuration/infinispan/DistributedCacheTestCase.java

```diff
@@ -0,0 +1,246 @@
+                .tab("Locking").rowName(CACHE_NAME).defineFirst("hasLocking").invoke();
+        checker.editSelectAndAssert(page, "isolation", "SERIALIZABLE").withVerifier(lockingVerifier)
+                .tab("Locking").rowName(CACHE_NAME).defineFirst("hasLocking").invoke();
+    }
+
+    @Test
+    @InSequence(14)
+    public void editStriping() {
+        checker.editCheckboxAndAssert(page, "striping", true).withVerifier(lockingVerifier)
+                .tab("Locking").rowName(CACHE_NAME).defineFirst("hasLocking").invoke();
+        checker.editCheckboxAndAssert(page, "striping", false).withVerifier(lockingVerifier)
+                .tab("Locking").rowName(CACHE_NAME).defineFirst("hasLocking").invoke();
+    }
+
+    @Test
+    @InSequence(15)
+    public void editAcquireTimeout() {
+        checker.editTextAndAssert(page, "acquireTimeout", "5000").withVerifier(lockingVerifier)
+                .tab("Locking").rowName(CACHE_NAME).defineFirst("hasLocking").invoke();
+        checker.editTextAndAssert(page, "acquireTimeout", "-1").expectError()
+                .tab("Locking").rowName(CACHE_NAME).defineFirst("hasLocking").invoke();
+    }
+
+    //TRANSACTION
+    @Test
+    @InSequence(16)
+    public void editMode() {
+        checker.editSelectAndAssert(page, "transactionMode", "NONE").withVerifier(transactionVerifier)
+                .tab("Trans").rowName(CACHE_NAME).defineFirst("hasTransaction").dmrAttribute("mode").invoke();
+        checker.editSelectAndAssert(page, "transactionMode", "NON_XA").withVerifier(transactionVerifier)
+                .tab("Trans").rowName(CACHE_NAME).defineFirst("hasTransaction").dmrAttribute("mode").invoke();
+    }
+
+    @Test
+    @InSequence(17)
+    public void editLocking() {
+        checker.editSelectAndAssert(page, "locking", "OPTIMISTIC").withVerifier(transactionVerifier)
+                .tab("Trans").rowName(CACHE_NAME).defineFirst("hasTransaction").invoke();
+        checker.editSelectAndAssert(page, "locking", "PESSIMISTIC").withVerifier(transactionVerifier)
+                .tab("Trans").rowName(CACHE_NAME).defineFirst("hasTransaction").invoke();
+    }
+
+    @Test
+    @InSequence(18)
+    public void editStopTimeout() {
+        checker.editTextAndAssert(page, "stopTimeout", "13400").withVerifier(transactionVerifier)
+                .tab("Trans").rowName(CACHE_NAME).defineFirst("hasTransaction").invoke();
+        checker.editTextAndAssert(page, "stopTimeout", "150asdf50").expectError()
+                .tab("Trans").rowName(CACHE_NAME).defineFirst("hasTransaction").invoke();
+        checker.editTextAndAssert(page, "stopTimeout", "-15000").expectError()
+                .tab("Trans").rowName(CACHE_NAME).defineFirst("hasTransaction").invoke();
+    }
+
+    @Test
+    @InSequence(19)
+    public void removeCache() {
+        page.content().getResourceManager().removeResourceAndConfirm(CACHE_NAME);
+    }
+}
```

Comment:
@InSequence(1)...@InSequence(19) Wow! Is such test dependency sequence really necessary or could it be redesigned?